src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -67,6 +67,8 @@ function AnchorRenderer({tnode, key, style}) {
 
     // An auth token is needed to download Expensify chat attachments
     const isAttachment = Boolean(htmlAttribs['data-expensify-source']);
+    const fileName = isAttachment && tnode.domNode && tnode.domNode.children && tnode.domNode.children[0] && tnode.domNode.children[0].data ? tnode.domNode.children[0].data : null;
```

Comment:
Let's use `lodashGet` to clean this up a bit

Reply:
Okay will do.

Reply:
Done